modules/document_repository/php/files.class.inc

```diff
@@ -205,7 +209,7 @@ class Files extends \NDB_Page
             $Notifier->notify($msg_data);
         }
 
-        $path = __DIR__ . "/../user_uploads/$dataDir";
+        $path = $path.$dataDir;
```

Comment:
I don't think it's safe to assume that path is defined with a `/` at the end of it (same for the other places the filename is concatenated to it.) It should explicitly add it (and throw an exception or otherwise die beforehand it path is empty for security reasons.)

Reply:
I just followed the assumption made in other uploading forms. If i make changes to this one do i make it to the others too ?

Reply:
you mean other forms in this module or other forms in LORIS?

Reply:
Loris. Like media for example

Reply:
no, just fix these ones.

Reply:
@ridz1208 If you don't mind it might be useful to do this using a Utility function. 

e.g.

`\Utility::AppendFrontSlash(string $path)` that adds a front slash to the end of a string unless one already exists. Then we can easily use it throughout LORIS.

Reply:
I was actually gonna look for the function that validates paths on config module. We could reuse that code to validate before upload too no ?

Reply:
Hmm we could temporarily I suppose. A better solution would be to rework modules to use the new File Uploader class because that has all of that validation built in. But that's a more substantial change.

Reply:
I think it would be a good idea to more consistently handle this with something like `os.path.join` (Python) or `filepath.Join` (Go), but that's beyond the scope of this PR.

Reply:
Yeah I think I'm agreeing with you just doing the PHP equivalent. I think this works (adapted from some WordPress code):

```php
    return rtrim( $string, '/\\' ) . '/';
```

We could wrap it in a Utility function just so that we have a standard way of doing this. not necessary though.

Reply:
Lets talk after i have questions

Reply:
As far as I know PHP doesn't have a built in equivalent. I think we should write one with appropriate unit tests, but I don't think this PR is the right place to do it.